lib/improver/utilities/pad_spatial.py

```diff
@@ -125,6 +128,44 @@ def create_cube_with_halo(cube, halo_radius):
     return halo_cube
 
 
+def remove_cube_halo(cube, halo_radius):
+    """
+    Remove halo of halo_radius from a cube.
```

Comment:
Can you explain why this is needed here - ie to remove a spatial halo from a cube with more than 2 dimensions.  It might also make sense to place this with `remove_halo_from_cube` in the file, so that the relationship between the two is clearer (otherwise it's easy to think this function is a duplicate).

Reply:
Added